ee_tests/src/booster_ee_int_tests/booster_trigger_cd.spec.ts

```diff
@@ -10,10 +9,31 @@ import { SpaceDashboardPage } from '../page_objects/space_dashboard.page';
 import { SpacePipelinePage } from '../page_objects/space_pipeline.page';
 import { MainDashboardPage } from '../page_objects/main_dashboard.page';
 import { StageRunPage } from '../page_objects/space_stage_run.page';
+import { SpaceChePage } from '../page_objects/space_che.page';
+import { BoosterEndpoint } from '../page_objects/booster_endpoint.page';
+import { SpaceCheWorkspacePage } from '../page_objects/space_cheworkspace.page';
 import { info } from '../support';
+import * as ui from '../ui';
 
 let globalSpaceName: string;
 let globalSpacePipelinePage: SpacePipelinePage;
+const SRCFILENAME: string = 'README.adoc';
+
+/* Text used to verify operation of deployed app, before and after the app is modified */
+const EXPECTED_TEXT_AFTER_SEND = 'Howdee, World';
+const EXPECTED_TEXT_AFTER_RECEIVED = 'Howdee, Howdee, World!';
+const EXPECTED_SUCCESS_TEXT = 'Succeeded in deploying verticle';  // TODO - Need unique string for each booster
```

Comment:
The unique string for each booster (runtime) is already [in the code](https://github.com/fabric8io/fabric8-test/blob/master/ee_tests/src/support/launcher_runtime.ts#L40). You can get it from quickstart:
```
const EXPECTED_SUCCESS_TEXT = new Quickstart(browser.params.quickstart.name).runtime.quickstartStartedTerminal;
```